### PR TITLE
PMM-7339 Allow to start checks individually

### PR DESCRIPTION
**Pull request by artemgavrilov**
[PMM-7339](https://jira.percona.com/browse/PMM-7339)

- [ ] https://github.com/percona/pmm-managed/pull/683

### CODE REVIEW

**Review by Dasio — APPROVED** [review]

File: api/managementpb/checks.proto [hunk 1]
```diff
@@ -74,6 +78,9 @@ service SecurityChecks {
       post: "/v1/management/SecurityChecks/Start"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "If no checks names specified all available checks will be started."
```
Comment (oter) [inline api/managementpb/checks.proto:82]: minor: better to say `All the available checks will be started if check names weren't specified.`

**Review by oter — APPROVED** [review]

**Review by palash25 — APPROVED** [review]

File: api/managementpb/checks.proto (hunk 1 above)
Comment (artemgavrilov) [reply api/managementpb/checks.proto]: Done

File: api/swagger/swagger.json [hunk 2]
```diff
@@ -12609,13 +12642,16 @@
...
...shalAny(any, foo); err != nil {\n       ...\n     }\n\nThe pack methods provided by protobuf library will by default use\n'type.googleapis.com/full.type.name' as the type URL and the unpack\nmethods only use the fully qualified type name after the last '/'\nin the type URL, for example \"foo.bar.com/x/y.z\" will yield type\nname \"y.z\".\n\n\nJSON\n====\nThe JSON representation of an `Any` value uses the regular\nrepresentation of the deserialized, embedded message, with an\nadditional field `@type` which contains the type URL. Example:\n\n    package google.profile;\n    message Person {\n      string first_name = 1;\n      string last_name = 2;\n    }\n\n    {\n      \"@type\": \"type.googleapis.com/google.profile.Person\",\n      \"firstName\": \u003cstring\u003e,\n      \"lastName\": \u003cstring\u003e\n    }\n\nIf the embedded message type is well-known and has a custom JSON\nrepresentation, that representation will be embedded adding a field\n`value` which holds the custom JSON in addition to the `@type`\nfield. Example (for message [google.protobuf.Duration][]):\n\n    {\n      \"@type\": \"type.googleapis.com/google.protobuf.Duration\",\n      \"value\": \"1.212s\"\n    }",
```
Comment (BupycHuk) [inline api/swagger/swagger.json:12645]: Do we really need so much repeated description in our swagger.json? Is it possible to not show description if it's not set by default?

Comment (artemgavrilov) [reply api/swagger/swagger.json:12645]: Oh gosh, I can't find a way to disable it. These descriptions enabled by adding `annotations.proto` import, once grpc-gateway detects such import it starts adding default descriptions.

File: api/managementpb/checks.proto [hunk 3]
```diff
@@ -74,6 +78,9 @@ service SecurityChecks {
       post: "/v1/management/SecurityChecks/Start"
       body: "*"
     };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      description: "All the available checks will be started if check names weren't specified."
```
Comment (atymchuk) [inline api/managementpb/checks.proto:82]:
```suggestion
      description: "All the available checks will be started if check names aren't specified."
```

**Review by atymchuk — CHANGES_REQUESTED** [review]

**Review by atymchuk — APPROVED** [review]

**Review by palash25 — APPROVED** [review]

**Pull request merged**